### ISSUE TITLE
Widget profile

### ISSUE THREAD

**Issue by Matt0550** [bprayush/school-ui-toolkit#13]
Where only the guides for this widget?
How can I recreate it?
![Immagine 2021-09-26 202712](https://user-images.githubusercontent.com/65899974/134819715-38355ee9-883c-4c7b-a601-636aeb252edd.jpg)

**Comment by bugthedebugger** [comment]
You use different provided widgets to make that screen.

**Comment by Matt0550** [comment]
> You use different provided widgets to make that screen. 

Yes, I know, but how can I insert the button inside the profile card, for example, or how can I add other fields to the profile card

**Comment by Matt0550** [comment]
?
